docs/documentation/upgrading/topics/changes/changes-26_3_1.adoc

```diff
@@ -0,0 +1,9 @@
+// ------------------------ Notable changes ------------------------ //
+== Notable changes
+
+Notable changes where an internal behavior changed to prevent common misconfigurations, fix bugs or simplify running {project_name}.
+
+=== Options for additional datasources marked as preview
+
+Options provided for settings of additional datasources have been marked as preview, as further changes might be added in the next releases.
+Unfortunately, these options have not been marked for the {project_name} 26.3.0 major release as they should.
```

Comment:
Maybe we could say something like this to be more clear?
```suggestion
In the 26.3.0 release, the newly added options for configuring additional datasources were missing a preview label. This has been now corrected as the work on this feature continues over a few next releases.
```
WDYT?

Reply:
Works for me, thanks!